sql/hive/src/main/scala/org/apache/spark/sql/hive/client/HiveClientImpl.scala

```diff
@@ -378,6 +380,47 @@ private[hive] class HiveClientImpl(
 
       val properties = Option(h.getParameters).map(_.asScala.toMap).orNull
 
+      val excludedTableProperties = Set(
+        // The following are hive-generated statistics fields. Currently, only total_size and
+        // row_count are used to populate the dedicated field `stats`.
+        // TODO: stats should include all the other three fields.
+        StatsSetupConst.COLUMN_STATS_ACCURATE,
+        StatsSetupConst.NUM_FILES,
+        StatsSetupConst.NUM_PARTITIONS,
+        StatsSetupConst.ROW_COUNT,
+        StatsSetupConst.RAW_DATA_SIZE,
```

Comment:
That means, we will overwrite the Hive-generated statistics `TOTAL_SIZE` and `RAW_DATA_SIZE` by our statistics. This could be a surprise to the users who are using both Hive and Spark on the same data sets, when they issues an alter table from Spark.

If we want to do not hide the other Hive-specific fields (e.g., `NUM_FILES`, `NUM_PARTITIONS`), `SHOW CREATE TABLE` needs to explicitly exlude them, like what we did in the PR: https://github.com/apache/spark/pull/14855. Does this sound OK?